frontend/src/components/dialogs/ProjectRequestersDialog.js

```diff
@@ -0,0 +1,178 @@
+import {
+  Avatar,
+  Button,
+  Container,
+  Divider,
+  IconButton,
+  LinearProgress,
+  Link,
+  makeStyles,
+  Table,
+  TableBody,
+  TableCell,
+  TableRow,
+  Typography,
+} from "@material-ui/core";
+import BlockIcon from "@material-ui/icons/Block";
+import CheckIcon from "@material-ui/icons/Check";
+import React, { useContext } from "react";
+
+// Relative imports
+import { getLocalePrefix } from "../../../public/lib/apiOperations";
+import { getImageUrl } from "../../../public/lib/imageOperations";
+import getTexts from "../../../public/texts/texts";
+import UserContext from "../context/UserContext";
+import GenericDialog from "./GenericDialog";
+
+const useStyles = makeStyles((theme) => ({
+  user: {
+    display: "flex",
+    alignItems: "center",
+  },
+  avatar: {
+    marginRight: theme.spacing(1),
+  },
+  username: {
+    fontWeight: 600,
+  },
+  followedText: {
+    [theme.breakpoints.down("xs")]: {
+      fontSize: 13,
+    },
+  },
+  loginButton: {
+    marginTop: theme.spacing(3),
+  },
+  loginButtonContainer: {
+    display: "flex",
+    justifyContent: "center",
+  },
+}));
+
+export default function ProjectRequestersDialog({
+  loading,
+  onClose,
+  open,
+  project,
+  requesters,
+  url,
+  user,
+}) {
+  const classes = useStyles();
+  const { locale } = useContext(UserContext);
+  const texts = getTexts({ page: "project", locale: locale });
+
+  const handleClose = () => {
+    onClose();
+  };
+
+  return (
+    <GenericDialog
+      onClose={handleClose}
+      open={open}
+      // TODO(Piper): iterate on naming here
+      // title={texts.requesters_of + " " + project.name}
+      title={"Requested to join project"}
+      //   title={"Users who have requested to follow"}
```

Comment:
What title makes the most sense here?

Reply:
`Members requested to join project` makes sense to me